utils/profile-services.ts

```diff
@@ -0,0 +1,185 @@
+import { LimitedProfile } from "@pagopa/io-functions-commons/dist/generated/definitions/LimitedProfile";
+import { RetrievedProfile } from "@pagopa/io-functions-commons/dist/src/models/profile";
+import { ServiceId } from "@pagopa/io-functions-commons/dist/generated/definitions/ServiceId";
+import {
+  makeServicesPreferencesDocumentId,
+  RetrievedServicePreference,
+  ServicesPreferencesModel
+} from "@pagopa/io-functions-commons/dist/src/models/service_preference";
+import {
+  IResponseErrorNotFound,
+  ResponseErrorNotFound
+} from "italia-ts-commons/lib/responses";
+import * as e from "fp-ts/lib/Either";
+import * as te from "fp-ts/lib/TaskEither";
+import * as t from "fp-ts/lib/Task";
+import { ServicesPreferencesModeEnum } from "@pagopa/io-functions-commons/dist/generated/definitions/ServicesPreferencesMode";
+import * as a from "fp-ts/lib/Array";
+import { BlockedInboxOrChannelEnum } from "@pagopa/io-functions-commons/dist/generated/definitions/BlockedInboxOrChannel";
+import { TaskEither } from "fp-ts/lib/TaskEither";
+import { NonNegativeInteger } from "italia-ts-commons/lib/numbers";
+
+const NOT_FOUND_TITLE = "ServicesPreferences not found";
+
+interface IServicePreferenceHandler {
+  readonly isMyReposability: (profile: RetrievedProfile) => boolean;
```

Comment:
@balanza suggested on my branch this:
<img src="https://user-images.githubusercontent.com/265564/124590433-6d2d2f00-de5b-11eb-882c-c77ccf2baa5e.jpeg">